binderhub/build.py

```diff
@@ -40,13 +40,16 @@ class BuildStatus(Enum):
         The state the build is now in
 
         Used when `kind` is `Kind.BUILD_STATUS_CHANGE`
+
+        These enum values are referenced in the front-end to display the build status,
+        hence the mismatch between the Enum name (backend) and value (frontend).
         """
 
-        PENDING = 1
-        RUNNING = 2
-        COMPLETED = 3
-        FAILED = 4
-        UNKNOWN = 5
+        PENDING = "pending"
+        RUNNING = "running"
+        COMPLETED = "built"
```

Comment:
This mismatch in naming is due to
https://github.com/jupyterhub/binderhub/blob/c4af713bb20ed90606e9795834d160bd8a332578/binderhub/builder.py#L515-L517
Do you think we should rename the enum to `BUILT`?

Reply:
Probably less confusing that way, sure.